src/modules/map/components/openlayers/utils/useMapInteractions.composable.js

```diff
@@ -182,8 +182,9 @@ export default function useMapInteractions(map) {
         })
     }
 
-    function onMapRightClick(event) {
+    function onMapRightClick(event, updateLongClickTriggered = false) {
         clearTimeout(longClickTimeout)
+        longClickTriggered = updateLongClickTriggered
```

Comment:
Why not set the value to `false` here? Because I don't see the `onMapRightClick` function is called with different value for `updateLongClickTriggered`.

Reply:
Well, it was meant to be set to true in the `longClickTimeout` call but I somehow forgot to add the argument, thanks for pointing this out.